pkg/lint/rules/template.go

```diff
@@ -106,11 +106,9 @@ func Templates(linter *support.Linter, values []byte, namespace string, strict b
 			continue
 		}
 
-		// NOTE: disabled for now, Refs https://github.com/kubernetes/helm/issues/1463
 		// Check that all the templates have a matching value
```

Comment:
Can you remove the comment and commented out function call?

Reply:
Which function call needs to be removed?

Reply:
the one on line 110.